static/app/components/events/interfaces/ai-monitoring/aiMonitoringSection.tsx

```diff
@@ -0,0 +1,77 @@
+import Alert from 'sentry/components/alert';
+import {LinkButton} from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import {EventDataSection} from 'sentry/components/events/eventDataSection';
+import PageFiltersContainer from 'sentry/components/organizations/pageFilters/container';
+import {IconOpen} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import type {Event} from 'sentry/types/event';
+import type {Organization} from 'sentry/types/organization';
+import type {Project} from 'sentry/types/project';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {
+  NumberOfPipelinesChart,
+  TotalTokensUsedChart,
+} from 'sentry/views/aiMonitoring/aiMonitoringCharts';
+import * as ModuleLayout from 'sentry/views/performance/moduleLayout';
+import {useIndexedSpans} from 'sentry/views/starfish/queries/useIndexedSpans';
+import {type IndexedResponse, SpanIndexedField} from 'sentry/views/starfish/types';
+
+interface Props {
+  event: Event;
+  organization: Organization;
+  project: Project;
+}
+
+export default function AIMonitoringSection({event, organization, project}: Props) {
+  const traceId = event.contexts.trace?.trace_id;
+  const spanId = event.contexts.trace?.span_id;
+  const {data, error, isLoading} = useIndexedSpans({
+    limit: 1,
+    fields: [SpanIndexedField.SPAN_AI_PIPELINE_GROUP],
+    referrer: 'api.ai-pipelines.view',
+    search: new MutableSearch(`trace:${traceId} id:"${spanId}"`),
+  });
+  const aiPipelineGroup =
+    data && (data[0] as IndexedResponse)?.[SpanIndexedField.SPAN_AI_PIPELINE_GROUP];
+
+  const actions = (
+    <ButtonBar gap={1}>
+      <LinkButton
+        size="xs"
+        icon={<IconOpen />}
+        to={`/organizations/${organization.slug}/ai-monitoring/`}
+      >
+        {t('View in AI Monitoring')}
+      </LinkButton>
+    </ButtonBar>
+  );
+
+  return (
+    <EventDataSection
+      title={t('AI monitoring')}
+      type="ai-monitoring"
+      help={t('Charts showing how many tokens are being used')}
+      actions={actions}
+    >
+      <PageFiltersContainer forceProject={project} shouldForceProject>
```

Comment:
i think we probably don't want to insert this into a sub component that loads only sometimes. we usually only render this in views. I'd try to insert this somewhere higher?

Reply:
you might have to play around with putting this in group details somewhere static/app/views/issueDetails/groupDetails.tsx

Reply:
I removed page filters container for now - the way graphs are scoped to particular projects or time periods is different for every team right now, we should probably make an effort to standardize that at some point. (DDM, performance, and issues all have different graph components and time picker UIs)